website/app.js

```diff
@@ -154,7 +154,7 @@ function generate(
     size: {
       height: 300,
       // @ts-ignore
```

Comment:
This can be removed now with the below change.

Reply:
I removed it just now, but it resulted in a red CI.